### PR TITLE
CMCL-0000: Fix hdrp compilation errors

### PR DESCRIPTION
**Pull request by AntoineCharton**
Fix HDRP test compiler errors and move them into the main package test.

### CODE REVIEW

File: com.unity.cinemachine/Tests/Editor/Unity.Cinemachine.Editor.Tests.asmdef [hunk 1]
```diff
@@ -21,6 +21,12 @@
     "defineConstraints": [
         "UNITY_INCLUDE_TESTS"
     ],
-    "versionDefines": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.render-pipelines.high-definition",
+            "expression": "14.0.0",
+            "define": "TEST_CINEMACHINE_HDRP"
```
Comment (glabute) [inline com.unity.cinemachine/Tests/Editor/Unity.Cinemachine.Editor.Tests.asmdef:28]: I think the define should be CINEMACHINE_HDRP, for consistency with the rest of CM

File: com.unity.cinemachine/Tests/Runtime/HDRP/CheckPhysicalCameraProperties.cs [hunk 2]
```diff
@@ -0,0 +1,106 @@
+#if TEST_CINEMACHINE_HDRP
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Unity.Cinemachine.Tests.HDRP
```
Comment (glabute) [inline com.unity.cinemachine/Tests/Runtime/HDRP/CheckPhysicalCameraProperties.cs:7]: We don't want a separate HDRP folder.  Just put it in Runtime, and adjust the namespace to match the other things in Runtime

**Review by glabute — CHANGES_REQUESTED** [review]

File: com.unity.cinemachine/Tests/Editor/Unity.Cinemachine.Editor.Tests.asmdef (hunk 1 above)
Comment (glabute) [reply com.unity.cinemachine/Tests/Editor/Unity.Cinemachine.Editor.Tests.asmdef:28]: @AntoineCharton Please address this comment

**Review by glabute — CHANGES_REQUESTED** [review]

File: com.unity.cinemachine/Tests/Runtime/HDRP/CheckPhysicalCameraProperties.cs (hunk 2 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Tests/Runtime/HDRP/CheckPhysicalCameraProperties.cs]: Fixed

**Review by glabute — APPROVED** [review]

**Pull request merged**